Cell Cell communication/views/Parts and Devices.csv

```diff
@@ -0,0 +1,39 @@
+,,,,,,,,,,,,
+,Blue text column headers are optional,,,,,,,,,,,
+DNA Parts and Devices,Click on column headers for more information as to what to put into the column,,,,,,,,,,,
+"Short and human friendly name for the part, e.g., ""GFP Reporter""",This is the role of the part according to the sequence ontology (see Ontology tab for full list). Select from the drop down menu or type have it validated.,Comments about design decisions. E.g. GC content or restriction sites removed,"If no changes were made use Direct, otherwise say changes that were made (can choose for drop down menu)",A description of how the part is meant to function and any information about the part that is not to do with how it was designed.,Where is the information about this part coming from? Provide a tag and e.g. AG for an Addgene plasmid. Tags found in the Source Sheet,Where is the information about this part coming from? Provide a te.g. 87906 for an Addgene plasmid. This is interpreted together with the column to the left,"Where is the genetic information coming from, e.g. gene taken from Aequorea victoria if available add in strain name too. This should match the scientific name for the txid in the next column","Where will the part be used, e.g. construct for use in Escherichia coli 98NK2. Where available include the strain name. This should match the scientific name for the txid in the next column","True if this is a final product to be produced, false if this is an intermediate stage being specified for compactness",Whether or not the part is circular. Select TRUE (circular) or FALSE (linear).,Automatically calculated length of the sequence pasted to the right. Use this column for checking there are no paste errors.,The series of ATGCs that make up the part. Spaces can be used within the sequence but no other characters.
+Part Name,Role,Design Notes,Altered Sequence,Part Description,Data Source Prefix,Data Source ID,Source Organism,Target Organism,Final Product,Circular,length (bp),Sequence
+pSB1C3,plasmid,,,,iGEM registry,pSB1C3,,Escherichia coli,False,True,0,
+B0015,,,,,iGEM registry,B0015,,,False,False,0,
+B0030,,,,,iGEM registry,B0030,,,False,False,0,
+B0034,,,,,iGEM registry,B0034,,,False,False,0,
+B1006,,,,,iGEM registry,B1006,,,False,False,0,
+C0061,,,,,iGEM registry,C0061,,,False,False,0,
+C0062,,,,,iGEM registry,C0062,,,False,False,0,
+C0078,,,,,iGEM registry,C0078,,,False,False,0,
+C0179,,,,,iGEM registry,C0179,,,False,False,0,
+E0040,,,,,iGEM registry,E0040,,,False,False,0,
+J23100,,,,,iGEM registry,J23100,,,False,False,0,
+K649000,,,,,iGEM registry,K649000,,,False,False,0,
+K934024,,,,,iGEM registry,K934024,,,False,False,0,
+R0040,,,,,iGEM registry,R0040,,,False,False,0,
+R0062,,,,,iGEM registry,R0062,,,False,False,0,
+pSB3K3,plasmid,,,,iGEM registry,pSB3K3,,,False,True,0,
+pSB6A1,plasmid,,,,iGEM registry,pSB6A1,,,False,True,0,

```

Comment:
@vinoo-igem Are pSB3K3 and pSB6A1 vectors that we are allowed to use this year?

Reply:
We won't have those available.

Reply:
Thanks for the comment. I have removed plasmids other than pSB1C3. Is there any suitable column in csv files to keep records of backbones used in previous experiments?

Reply:
Design notes and part description are both reasonable places to put such information.